src/components/20-molecules/dropdown/react/DemoDropdownReact.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import AXADropdownReact from './AXADropdownReact';
+import AXACheckboxReact from '../../../10-atoms/checkbox/react/AXACheckboxReact';
+
+const DemoDropdown = () => {
```

Comment:
As mentioned before, I would only have the wrapper in here and do all the rest with those arrays and properties outside of here, if possible. Because the user won't use the wrapper like this, he will keep it generic.

Reply:
See my feedback before.